app/components/public/speaker-item.js

```diff
@@ -1,7 +1,26 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed } = Ember;
 
 export default Component.extend({
-  classNames: ['column', 'four', 'wide']
+  classNameBindings: ['style'],
+
+  style: computed(function() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      return 'column eight wide';
+    } else {
+      return 'column four wide';
+    }
+  }),
+  didInsertElement() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      this.$('.speaker-item').addClass(' ui grid stackable');
```

Comment:
@geekyd you can use if statements in templates too. 

`<div class="{{if speaker.isFeatured 'class-one' 'class-two'}}"></div>'